src/main/java/edu/kis/powp/jobs2d/drivers/adapter/Job2dDriverAdapter.java

```diff
@@ -0,0 +1,49 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.legacy.drawer.shape.LineFactory;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.features.DrawerFeature;
+
+import java.awt.*;
+
+/**
+ * driver adapter to drawer with several bugs.
+ */
+public class Job2dDriverAdapter extends DrawPanelController implements Job2dDriver {
```

Comment:
The name is better but it does not indicate the purpose of the adapter or what was actually adapted.

Inheritance is not used (no super drawLine calls). Should be removed!